python/amici/petab_objective.py

```diff
@@ -303,8 +301,8 @@ def create_parameter_mapping_for_condition(
     :param parameter_mapping_for_condition:
         Preliminary parameter mapping for condition.
     :param condition:
-        pandas.DataFrame row with preequilibrationConditionId and
-        simulationConditionId.
+        :class:`pandas.DataFrame` row with ``preequilibrationConditionId`` and
```

Comment:
`pandas` links seem to work in docs, but `petab` links don't (e.g. see type hint for `petab_problem` in the docs)

https://amici--1817.org.readthedocs.build/en/1817/generated/amici.petab_objective.html#amici.petab_objective.create_parameter_mapping_for_condition

Reply:
Yes, I noticed that as well (#1818). Will check separately.